### ISSUE TITLE
[Bug]: (accessibility) toggle button not accessible n Aquatic and Desert mode

### ISSUE THREAD

**Issue by tanneruzzell-msft** [microsoft/fluentui#34360]
### Component

ToggleButton

### Package version

9.3.86

### React version

17.0.2

### Environment

```shell
System:
    OS: Windows 11 10.0.26100
    CPU: (12) x64 13th Gen Intel(R) Core(TM) i7-1355U
    Memory: 2.02 GB / 15.68 GB
  Browsers:
    Edge: Chromium (135.0.3179.66)
  npmPackages:
    @fluentui/merge-styles: 8.6.4 => 8.6.4
    @fluentui/react: 8.118.2 => 8.118.2
    @fluentui/react-accordion: 9.4.3 => 9.4.3
    @fluentui/react-avatar: 9.6.32 => 9.6.32
    @fluentui/react-badge: 9.2.40 => 9.2.40
    @fluentui/react-button: 9.3.95 => 9.3.95
    @fluentui/react-card: 9.0.86 => 9.0.86
    @fluentui/react-carousel: 9.3.1 => 9.3.1
    @fluentui/react-charting: 5.21.7 => 5.21.7
    @fluentui/react-checkbox: 9.2.31 => 9.2.31
    @fluentui/react-combobox: 9.13.1 => 9.13.1
    @fluentui/react-dialog: 9.11.5 => 9.11.5
    @fluentui/react-divider: 9.2.72 => 9.2.72
    @fluentui/react-drawer: 9.5.5 => 9.5.5
    @fluentui/react-field: 9.2.0 => 9.2.0
    @fluentui/react-hooks: 8.8.1 => 8.8.1
    @fluentui/react-icons: 2.0.245 => 2.0.245
    @fluentui/react-image: 9.1.70 => 9.1.70
    @fluentui/react-infolabel: 9.0.56 => 9.0.56
    @fluentui/react-input: 9.4.82 => 9.4.82
    @fluentui/react-jsx-runtime: 9.0.53 => 9.0.53
    @fluentui/react-link: 9.2.27 => 9.2.27
    @fluentui/react-menu: 9.14.10 => 9.14.10
    @fluentui/react-message-bar: 9.2.5 => 9.2.5
    @fluentui/react-persona: 9.2.91 => 9.2.91
    @fluentui/react-popover: 9.9.31 => 9.9.31
    @fluentui/react-progress: 9.1.81 => 9.1.81
    @fluentui/react-search: 9.0.11 => 9.0.11
    @fluentui/react-select: 9.1.81 => 9.1.81
    @fluentui/react-shared-contexts: 9.23.1 => 9.23.1
    @fluentui/react-skeleton: 9.1.9 => 9.1.9
    @fluentui/react-slider: 9.3.0 => 9.3.0
    @fluentui/react-spinner: 9.4.11 => 9.4.11
    @fluentui/react-switch: 9.1.88 => 9.1.88
    @fluentui/react-table: 9.15.10 => 9.15.10
    @fluentui/react-tabs: 9.6.0 => 9.6.0
    @fluentui/react-tabster: 9.24.0 => 9.24.0
    @fluentui/react-tag-picker: 9.2.2 => 9.2.2
    @fluentui/react-tags: 9.3.11 => 9.3.11
    @fluentui/react-teaching-popover: 9.1.20 => 9.1.20
    @fluentui/react-text: 9.4.22 => 9.4.22
    @fluentui/react-textarea: 9.3.82 => 9.3.82
    @fluentui/react-theme: 9.1.24 => 9.1.24
    @fluentui/react-tooltip: 9.4.33 => 9.4.33
    @fluentui/react-tree: 9.7.4 => 9.7.4
    @fluentui/react-utilities: 9.18.23 => 9.18.23
    @fluentui/style-utilities: 8.10.9 => 8.10.9
    @types/react: 17.0.47 => 17.0.47
    @types/react-dom: 17.0.17 => 17.0.17
    react: 17.0.2 => 17.0.2
    react-dom: 17.0.2 => 17.0.2
```

### Current Behavior

When in Aquatic and Desert mode contrast ratio for toggleButton is  1.458:1 & 1.272:1 which  is less than 3:1, which is requried by microsoft a11y team

### Expected Behavior

Higher contrast for all button appearances. 

### Reproduction

https://react.fluentui.dev/?path=/docs/components-button-togglebutton--docs

### Steps to reproduce

Go to: https://react.fluentui.dev/?path=/docs/components-button-togglebutton--docs
Change computer display to aquatic or subtle mode. 
Go to appearence tab and click "subtle" 
see issue

![Image](https://github.com/user-attachments/assets/4032b39d-92b8-462b-88cd-acdbf14a07af)

### Are you reporting an Accessibility issue?

yes

### Suggested severity

Medium - Has workaround

### Products/sites affected

https://react.fluentui.dev/?path=/docs/components-button-togglebutton--docs

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by CampbellOwen** [comment]
Hi @tanneruzzell-msft, can you elaborate on the contrast issue? I tried using Aquatic on a checked ToggleButton and seem to have 3.03:1 contrast ratio:

![Image](https://github.com/user-attachments/assets/1aa58e99-01fa-4c45-840c-00071752057e)

![Image](https://github.com/user-attachments/assets/411bbeb4-0826-4a43-8d8c-88ed3473ab70)
https://webaim.org/resources/contrastchecker/?fcolor=313D50&bcolor=5C8E9C
